charts/trino/values.yaml

```
@@ -55,9 +55,21 @@ server:
     authenticationType: ""
     query:
       maxMemory: "4GB"
-  exchangeManager:
-    name: "filesystem"
-    baseDir: "/tmp/trino-local-file-system-exchange-manager"
+  exchangeManager: {}
+  # exchangeManager -- Mandatory [exchange manager
```

If you include the property name before `--`, it needs to be fully qualified:
```suggestion
  # server.exchangeManager -- Mandatory [exchange manager
```